lib/modules/datasource/terraform-provider/index.ts

```diff
@@ -79,24 +79,14 @@ export class TerraformProviderDatasource extends TerraformDatasource {
     const serviceDiscovery = await this.getTerraformServiceDiscoveryResult(
       registryURL
     );
-    const backendURL = `${registryURL}${serviceDiscovery['providers.v1']}${repository}`;
-    const res = (await this.http.getJson<TerraformProvider>(backendURL)).body;
+    const backendURL = `${registryURL}${serviceDiscovery['providers.v1']}${repository}/versions`;
+    const res = (await this.http.getJson<TerraformProviderVersions>(backendURL))
+      .body;
     const dep: ReleaseResult = {
-      releases: res.versions.map((version) => ({
+      releases: res.versions.map(({ version }) => ({
         version,
       })),
     };
-    if (res.source) {
-      dep.sourceUrl = res.source;
-    }
-    // set published date for latest release
-    const latestVersion = dep.releases.find(
-      (release) => res.version === release.version
-    );
-    // istanbul ignore else
-    if (latestVersion) {
-      latestVersion.releaseTimestamp = res.published_at;
-    }
```

Comment:
Isn't this information available at some place?

Reply:
same as above, these APIs are minimal, e.g. https://www.terraform.io/internals/provider-registry-protocol#sample-response